code/chart.js

```diff
@@ -1,4 +1,37 @@
-//DOM-selector for the canvas 👇
-const ctx = document.getElementById('chart').getContext('2d')
 
-//"Draw" the chart here 👇
+ // //DOM-selector for the canvas 👇
+const ctx = document.getElementById("chart").getContext("2d");
+
+
+const drawChart = (amount) => {
+const labels = [
+    'Projects done',
+    'Technigo projects',
+  ];
+
+  const data = {
+    labels: labels,
+    datasets: [{
+      label: 'My First dataset',
+      data: [amount, 19-amount],
```

Comment:
Great job passing in the amount of projected you have completed in your drawChart function. Nicely done!